<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpDialog</class>
 <widget class="QDialog" name="HelpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HelpDialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="itemListWidget"/>
   </item>
   <item>
    <widget class="QTableWidget" name="sourceTableWidget"/>
   </item>
   <item>
    <widget class="QTableWidget" name="productTableWidget"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
